### PR TITLE
feat- buttons and alerts

### PR DESCRIPTION
**Pull request by Irfan-vit**
This branch consists of all variations of buttons and alerts, also I have configured typography, colors, and

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/ae7ce8d210.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="./Components/Alert/alert.css">
+    <link rel="stylesheet" href="./Components/Button/button.css">
+</head>
+
+<body>
+    <!-- BBUTTONS -->
```
Comment (VishalPatil18) [inline index.html:16]: Typo in buttons spelling.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    <button class="btn" id="primary">primary button</button>
+    <button class="btn">secondary button</button>
+    <button class="btn" id="success">success button</button>
+    <button class="btn" id="warning">warning button</button>
+    <button class="btn" id="error">error button</button>
+    <button class="btn" disabled>disabled button</button>
+    <br>
+    <button class="btn large" id="default_btn">default button</button>
+    <button class="btn large" id="primary">primary button</button>
+    <button class="btn large">secondary button</button>
```
Comment (VishalPatil18) [inline index.html:27]: You've given ID to all other buttons but not this one. Ignore if you've done that intentionally or else you could fix that.

**Review by VishalPatil18 — APPROVED** [review]
A couple of fixes suggested. Apart from that, the code looks good. Great work Irfan 👍

File: index.html (hunk 2 above)
Comment (Irfan-vit) [reply index.html:27]: yep, that was intentional.

File: index.html (hunk 1 above)
Comment (Irfan-vit) [reply index.html:16]: 🙆‍♂️

File: index.html (hunk 2 above)
Comment (VishalPatil18) [reply index.html:27]: Great then, no need to change that 👍

**Pull request closed without merging**